demisto_sdk/commands/validate/tests/IF_validators_test.py

```diff
@@ -193,6 +208,143 @@ def test_IsCliNameReservedWordValidator_not_valid(reserved_word):
     )
 
 
+def test_IsFieldTypeChangedValidator_is_valid():
+    """
+    Given:
+        - IncidentFiled content items
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that the ValidationResult returned
+          for the IncidentField whose 'type' field has changed
+        - Ensure that no ValidationResult returned when 'type' field has not changed
+    """
+    old_type = "short text"
+    new_type = "html"
+
+    # not valid
+    content_item = create_incident_field_object(["type"], [new_type])
+    old_content_items = [create_incident_field_object(["type"], [old_type])]
+    create_old_file_pointers([content_item], old_content_items)
+    assert IsFieldTypeChangedValidator().is_valid([content_item])
+
+    # valid
+    content_item.field_type = old_type
+    assert not IsFieldTypeChangedValidator().is_valid([content_item])
+
+
+@pytest.mark.parametrize("unsearchable", (False, None))
+def test_UnsearchableKeyValidator_is_valid(unsearchable: bool):
+    """
+    Given:
+        - IncidentFiled content items
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that the ValidationResult returned
+          for the IncidentField whose 'unsearchable' field is set to false or not or undefined
+        - Ensure that no ValidationResult returned when unsearchable set to true
+    """
+    # not valid
+    content_item = create_incident_field_object(
+        paths=["unsearchable"], values=[unsearchable]
+    )
+    assert UnsearchableKeyValidator().is_valid([content_item])
+
+    # valid
+    content_item.unsearchable = True
+    assert not UnsearchableKeyValidator().is_valid([content_item])
+
+
+def test_NameFieldPrefixValidator_is_valid_without_item_prefix():
+    """
+    Given:
+        - IncidentField content items
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that the ValidationResult returned
+          for the IncidentField whose prefix name that not start with relevant pack name
```

Comment:
In case it is not valid, the name field of the `IncidentField` does not contain the name of the pack,
In the valid case, the name field of the `IncidentField` contains a prefix of the pack name field (`Foo`)